eth-contracts/contracts/service/registry/RegistryContract.sol

```diff
@@ -38,19 +38,15 @@ contract RegistryContract is InitializableV2, Ownable {
     }
 
     function kill() external {
-        requireIsInitialized();
-        assert (msg.sender == registryAddress);
-        selfdestruct(registryAddress);
-    }
-
-    /**
-     * @notice only allow storage contracts to be called by the respective factory.
-     *      i.e. TrackStorage methods can only be invoked by TrackFactory.
-     */
-    function onlyRegistrant(bytes32 _name) internal view {
+        _requireIsInitialized();
         require(
```

Comment:
added this require statement bc if the below require statement is hit with registryAddress = null it will break